egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -1281,6 +1281,7 @@ if ! "${skip_eval}"; then
         done
 
         [ -f local/score.sh ] && local/score.sh ${local_score_opts} "${asr_exp}"
+        [ -f local/score_swbd.sh ] && local/score_swbd.sh ${_data} ${_dir}/score_wer
```

Comment:
Why don't you create local/score.sh? Please not modify asr.sh as possible to invoke local scritps.

Reply:
Sure, I have fixed it now.

Reply:
Thank you!